modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -158,20 +236,33 @@ endif::[]
 Synchronization can sometimes take several hours.
 ====
 
+When you have the channels set up, associate each channel with the appropriate repository, and synchronize then channels (optionally configure scheduled syncronization).
+
+To do this, proceed to menu:Software[Manage > Channels], and for each channel you created, click on it and:
+
+. Navigate to the [guimenu]``Repositories``
+. Navigate to the [guimenu]``Sync``and click btn:[Sync Now] to begin synchronization immediately.
+. You can also setup a scheduled syncronization from this screen.
+
```

Comment:
See earlier edits (why is this repeated, though?)

Reply:
ditto